controller_manager:
  ros__parameters:
    update_rate: 200

    joint_state_broadcaster:
      type: "joint_state_broadcaster/JointStateBroadcaster"

    rl_quadruped_controller:
      type: "rl_quadruped_controller::RLQuadrupedController"
      policy_path: "/home/csl/rdog/test_ws/src/rl_quadruped_controller/pre_train/policy_him2.pt"
      config_path: "/home/csl/rdog/test_ws/src/rl_quadruped_controller/config/reddog_him.yaml"
